packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -130,17 +139,20 @@ export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX
     });
 
     /* @conditional-compile-remove(PSTN-calls) */
-    items.push({
-      key: 'showDialpadKey',
-      text: localeStrings.strings.call.openDtmfDialpad,
-      onClick: () => {
-        setShowDialpad(true);
-      },
-      iconProps: { iconName: 'Dialpad', styles: { root: { lineHeight: 0 } } },
-      itemProps: {
-        styles: buttonFlyoutIncreasedSizeStyles
-      }
-    });
+    // dtmf tone sending only works for 1:1 PSTN call
+    if (alternateCallerId && participantNumber <= 2) {
```

Comment:
This is incorrect. Even if the participant length is 1, we need to ensure that the participant is a PSTN number.
One could provide an alternate caller id to the composite and still call an acs user id.

Plus, I recommend using a function outside the component to do this calculation instead of doing it inline.
Something like `canShowDialpad()`